src/hw/getOnboardingStatus.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-bitwise */
+import Transport from "@ledgerhq/hw-transport";
+import getVersion from "./getVersion";
+import type { OnboardingInfo, SeedSize } from "../types/onboarding";
+
+// Cf https://ledgerhq.atlassian.net/wiki/spaces/FW/pages/3258089508/Synchronous+onboarding
+const onboardingFlag = 0x04;
+const recoveryModeFlag = 0x01;
+const seedRecoveryFlag = 0x80;
+const seedSizeFlag = 0x60;
+const currentWordFlag = 0x1f;
+const confirmingWordsFlag = 0x01;
+
+export default async function getOnboardingStatus(
+  transport: Transport
+): Promise<OnboardingInfo> {
+  const { flags } = await getVersion(transport);
```

Comment:
This should be working with supplied `getVersion` data, not calling `getVersion` again.